@xen-orchestra/web-core/lib/components/console/VtsRemoteConsole.vue

```diff
@@ -86,6 +86,16 @@ async function createVncConnection() {
   vncClient.addEventListener('connect', handleConnectionEvent)
 }
 
+watch(
+  () => uiStore.hasUi,
+  newVal => {
+    if (newVal && vncClient) {
+      if (vncClient.scaleViewport) vncClient.scaleViewport = false
+      vncClient.scaleViewport = true
+    }
+  }
+)
```

Comment:
What do you think of this:
```suggestion
whenever(
  () => uiStore.hasUi,
  () => {
    if (!vncClient) {
      return
    }

    vncClient.scaleViewport = false
    vncClient.scaleViewport = true
  }
)
```
I think it would be more readable.

Reply:
yes of course